releasenotes/notes/holland-vevn-9eb62140a2c8ff56.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - Holland now runs inside a python virtual environment
+    by default. By default, the virtual environment will
+    be placed in /openstack/venvs/holland-release-tag.
+upgrade:
+  - On upgrades, the holland pip packages will linger in 
+    the global environment if not cleaned up.
```

Comment:
what does this mean?  
Is this what used to happen? If so you need to be clear, and explain that this is now fixed and what the new behaviour is
Is this what happens now? If so, I wonder if we have the right solution...